pilot/pkg/config/kube/ior/ior_test.go

```diff
@@ -363,6 +363,11 @@ func TestEdit(t *testing.T) {
 			validateRoutes(t, c.hosts, list, "gw", c.tls)
 		})
 	}
+
+	// Try to edit a gateway with the same resourceVersion
+	// No effect. UPDATE Event is not generated. I'll remove it.
+	IORLog.SetOutputLevel(log.DebugLevel)
+	editGateway(t, store, "istio-system", "gw", []string{"one.org.br"}, map[string]string{"istio": "ingressgateway"}, true, "6")
```

Comment:
maybe we can replace this with a test that explicitly sends an event twice?

Reply:
done. It was not very straightforward but at least it's a start.